content/zh/blogs/kubesphere-3.1.0-ga-announcement.md

```diff
@@ -0,0 +1,135 @@
+
+## 更强大的可观测性
+
+可观测性是容器云平台非常关键的一环，狭义上主要包含监控、日志和追踪等，广义上还包括告警、事件、审计等。3.1.0 除了对已有的监控、日志、告警等功能进行优化升级，还新增了更多新特性。
+
++ 监控：支持图形化方式配置 `ServiceMonitor`，添加集群层级的自定义监控，同时还实现了类似于 Grafana 的 PromQL 语法高亮。
++ 告警：在 v3.1.0 进行了架构调整，不再使用 MySQL、Redis 和 etcd 等组件以及旧版告警规则格式，改为使用 Thanos Ruler 配合 Prometheus 内置告警规则进行告警管理，兼容 Prometheus 告警规则。
++ 通知管理：完成架构调整，与自研 Notification Manager v1.0.0 的全面集成，实现了以图形化界面的方式对接邮件、钉钉、企业微信、Slack、Webhook 等通知渠道。
++ 日志：新增了对 Loki 的支持，可以将日志输出到 [Loki](https://github.com/kubesphere/fluentbit-operator/blob/master/docs/plugins/output/loki.md)。还新增了对 kubelet/docker/containerd 的日志收集。
+
+![](https://cdn.jsdelivr.net/gh/yangchuansheng/imghosting@three/img/20210429140933.png)
+
+![](https://cdn.jsdelivr.net/gh/yangchuansheng/imghosting@three/img/20210429140844.png)
+
+<iframe width="760" height="380" src="//player.bilibili.com/player.html?aid=802847503&bvid=BV1iy4y1s7Vz&cid=330928839&page=1" scrolling="no" border="0" frameborder="no" framespacing="0" allowfullscreen="true"> </iframe>
+
+## 更易用的 DevOps
+
+KubeSphere 3.1.0 新增了 GitLab 多分支流水线和流水线克隆等功能，并内置了常用的流水线模板，帮助 DevOps 工程师提升 CI/CD 流水线的创建与运维效率。大部分场景下可基于流水线模板进行修改，不再需要从头开始创建，实现了真正的开箱即用。
+
+<iframe width="760" height="380" src="//player.bilibili.com/player.html?aid=845263571&bvid=BV1J54y1j7ev&cid=330928243&page=1" scrolling="no" border="0" frameborder="no" framespacing="0" allowfullscreen="true"> </iframe>
+
+<iframe width="760" height="380" src="//player.bilibili.com/player.html?aid=545285342&bvid=BV12q4y1E7qx&cid=330927777&page=1" scrolling="no" border="0" frameborder="no" framespacing="0" allowfullscreen="true"> </iframe>
+
+## 灵活可插拔的集群安装工具
+KubeKey 不仅支持 Kubernetes 1.17 ~ 1.20 在 AMD 64 与 ARM 64 的安装，还支持了 K3s。并且，Kubekey 还新增支持 Cilium、Kube-OVN 等网络插件。鉴于 Dockershim 在 K8s 1.20 中被废弃，Kubekey 可用于部署 containerd、CRI-O、iSula 等容器运行时，让用户按需快速创建集群。
+
+## 运维友好的网络管理
+
+KubeSphere 将 IaaS 云平台强大的网络能力继承到容器云平台，让用户在 Kubernetes 之上获得与 IaaS 一样稳定、安全和易用的网络使用体验。v3.1.0 新增了网络可视化拓扑图，你可以通过拓扑图洞悉各个服务之间的网络调用关系。
+
+鉴于 Calico 是目前最常用的 Kubernetes CNI 插件之一，v3.1.0 现已支持 Calico IP 池管理，也可以为 Deployment 指定静态 IP。此外，v3.1.0 还新增了对 [Kube-OVN](https://github.com/kubeovn/kube-ovn) 插件的支持。
+
+## 认证鉴权与多租户
+
+统一的身份管理和完备的鉴权体系，是多租户系统中实现逻辑隔离不可或缺的能力。KubeSphere 基于 Kubernetes 的角色访问控制（RBAC），提供了细粒度的权限控制能力，在支持 Namespace 层级资源隔离的同时通过 Workspace 进一步对租户进行了定义，多层级的权限管控更加契合企业用户的使用场景。
+
+v3.1.0 中新增了组织架构管理功能，可以通过用户组简化批量授权操作。除此之外还支持了企业空间的资源配额管理，实现对资源用量的管控，进一步满足企业用户的实际需求。
+
+统一认证方面，v3.1.0 中简化了身份提供商（IdentityProvider, IdP）的配置方式，除 LDAP 之外新增了对 CAS（Central Authentication Service）、OIDC（OpenID Connect）、OAuth2 等通用认证协议的支持，并提供了插件化的拓展方式，以便不同账户系统之间的集成。
+
+<iframe width="760" height="380" src="//player.bilibili.com/player.html?aid=802754514&bvid=BV1jy4y1s7i3&cid=331412102&page=1" scrolling="no" border="0" frameborder="no" framespacing="0" allowfullscreen="true"> </iframe>
+
+<iframe width="760" height="380" src="//player.bilibili.com/player.html?aid=972796031&bvid=BV1qp4y1b7Jz&cid=331391855&page=1" scrolling="no" border="0" frameborder="no" framespacing="0" allowfullscreen="true"> </iframe>
```

Comment:
![image](https://user-images.githubusercontent.com/1450685/116770725-51329900-aa78-11eb-9e25-08a2ee2ed394.png)

It looks like it does not work well here.